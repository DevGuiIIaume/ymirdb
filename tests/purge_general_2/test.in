set a 1
set b a 2
snapshot
del a
del b
del a
list entries
checkout 1
list entries
bye
