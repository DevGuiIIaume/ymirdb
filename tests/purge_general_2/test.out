> ok

> ok

> saved as snapshot 1

> not permitted

> ok

> ok

> no entries

> ok

> b [a 2]
a [1]

> bye
